### PR TITLE
Refactor strict hard toggle.

### PR DESCRIPTION
**Pull request by haldenl**
To merge after https://github.com/uwdata/draco/pull/82

### CODE REVIEW

**Comment by domoritz** [comment]
Can you update?

**Comment by haldenl** [comment]
Yeah, let me make a Draco release and then this should be okay.

**Comment by haldenl** [comment]
(good to go)

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "@types/json-stable-stringify": "^1.0.32",
+    "draco-core": "^0.0.6",
```
Comment (domoritz) [inline package.json]: Why do we add a new dependency?

Comment (haldenl) [reply package.json]: It should be there... not sure why it was missing in the first place.

**Pull request merged**